optimum/exporters/neuron/utils.py

```diff
@@ -92,22 +90,23 @@ def build_stable_diffusion_components_mandatory_shapes(
     vae_decoder_num_channels: Optional[int] = None,
     height: Optional[int] = None,
     width: Optional[int] = None,
+    num_images_per_prompt: Optional[int] = 1,
```

Comment:
are we storing the value in the config?

Reply:
No, in the end, we register the actual `batch_size` (`batch_size x num_images_per_prompt`) and infer directly the `num_images_per_prompt` by `batch_size` used for compiling unet or vae decoder divided by `batch_size` used for compiling text encoder or vae encoder. 

The reason is that I don't want `num_images_per_prompt` to increase the code complexity if it doesn't hurt user experience.